posthog/hogql/database/s3_table.py

```diff
@@ -45,7 +50,10 @@ def return_expr(expr: str) -> str:
         if structure:
             escaped_structure = add_param(structure, False)
 
-        expr = f"s3({escaped_url}"
+        if use_s3_cluster:
+            expr = f"s3Cluster('posthog', {escaped_url}"
```

Comment:
May worth to note: in EU `posthog` cluster does not include coordinators.
I think it's ok, we want to run it on workers only.